docs/zh/react/reactlynx-testing-library/index.mdx

```diff
@@ -1,20 +1,18 @@
-import { PackageManagerTabs } from '@theme';
-
 # ReactLynx Testing Library
 
-[ReactLynx Testing Library](https://github.com/lynx-family/lynx-stack/tree/main/packages/testing-library/react-lynx-testing-library) 基于 [Lynx Test Environment（`@lynx-js/test-environment`，Lynx 的 JS 实现）](./lynx-test-environment.md) 实现了一套 ReactLynx 组件的测试方案。
+ReactLynx Testing Library 借鉴了 [React Testing Library](https://testing-library.com/docs/react-testing-library/intro/) 的设计，适用于对于 ReactLynx 组件的单元测试。它屏蔽了 Lynx 双线程的实现细节，将 ReactLynx 的渲染流程抽象成 `render`、`fireEvent`、`screen` 等和 React Testing Library 相同的 API。和 React Testing Library 类似，用户也可以对页面中的 Element 进行断言，例如使用 `data-testid` 来快速寻找元素，使用 `@testing-library/jest-dom` 中的 `toBeInTheDocument` 判断元素在页面中。
 
-ReactLynx Testing Library 借鉴了 React Testing Library 的设计。底层的 Lynx Test Environment 负责提供 Lynx 环境的 JS 实现，类似于 `jsdom` 等工具提供的 DOM 环境。我们在 Lynx Test Environment 中使用了 `jsdom` 来实现 [Element PAPI](https://lynxjs.org/api/engine/element-api)，因此你可以直接使用 [`@testing-library/dom`](https://github.com/testing-library/dom-testing-library)、[`@testing-library/jest-dom`](https://github.com/testing-library/jest-dom) 等工具来辅助测试。
+## 配置
 
-## 适用场景
+### 从 create-rspeedy 创建新项目
 
-ReactLynx Testing Library 适用于对于 ReactLynx 组件的单元测试。它基于 Lynx Test Environment 和 `@testing-library/dom`（页面元素的查询和事件触发），屏蔽了 Lynx 双线程的实现细节，将 ReactLynx 的渲染流程抽象成 `render`、`fireEvent`、`screen` 等和 React Testing Library 类似的 API。用户也可以对页面中的 Element 进行断言，例如使用 `data-testid` 来快速寻找元素，使用 `@testing-library/jest-dom` 中的 `toBeInTheDocument` 判断元素在页面中。
+使用 `create-rspeedy` 创建的项目，在创建时可以主动选择是否使用 ReactLynx Testing Library（默认勾选），勾选后创建出来的项目已经配置好了 ReactLynx Testing Library。
 
-## 配置
+### 在已有项目中配置
 
-ReactLynx Testing Library 集成在 `@lynx-js/react` 包的 `testing-library` 子目录中，可以直接使用。
+ReactLynx Testing Library 集成在 `@lynx-js/react` 包的 `testing-library` 子目录中，无需额外安装其他包。
 
-配置 Vitest：
+配置 Vitest 时需要使用 `@lynx-js/react/testing-library/vitest-config` 中的 `createVitestConfig` 方法来创建 Vitest 配置。该方法会自动配置 [Lynx Testing Environment](https://github.com/lynx-family/lynx-stack/tree/main/packages/testing-library/testing-environment)，负责提供 Lynx 环境的 JS 实现。你可以通过 `mergeConfig` 方法将其和其他配置合并。
```

Comment:
Vitest need links

Reply:
We don't need to mention Lynx Testing Env here if we are going to change to mention it above.

You also shouldn't link to Github if you already have API reference of it in the website already